cli/search/search.go

```diff
@@ -115,6 +126,52 @@ func GetVersionsFromGitRemote(repo string) ([]version.Version, error) {
 	return versions, nil
 }
 
+// GetCommitsFromGitRemote returns commits list from specified remote git repo.
+func GetCommitsFromGitRemote(repo string) ([]string, error) {
+	commits := []string{}
+
+	if _, err := exec.LookPath("git"); err != nil {
+		return nil, fmt.Errorf("'git' is required for 'tt search' to work")
+	}
+
+	path, err := os.MkdirTemp("", "versions_log")
+
+	if err != nil {
+		return nil, fmt.Errorf("failed to get commits from %s: %s", repo, err)
+	}
+
+	os.Chmod(path, defaultDirPermissions)
```

Comment:
What are the default permissions, that we should change them?

Reply:
fixed